e2e/tests/apiproduct-crud.spec.ts

```
@@ -32,7 +32,7 @@ test.describe('APIProduct CRUD Operations', () => {
     }
   });
 
-  test('should navigate from list page to create page via Create button', { tag: '@smoke' }, async ({ page }) => {
+  test('should navigate from list page to create page via Create button', { tag: ['@smoke', '@api-management'] }, async ({ page }) => {
```

ESLint

[error] Replace `'should·navigate·from·list·page·to·create·page·via·Create·button',·{·tag:·['`@smoke`',·'`@api-management`']·},` with `⏎····'should·navigate·from·list·page·to·create·page·via·Create·button',⏎····{·tag:·['`@smoke`',·'`@api-management`']·},⏎···`

(prettier/prettier)